app/src/ui/lib/authentication-form.tsx

```diff
@@ -139,37 +138,38 @@ export class AuthenticationForm extends React.Component<
   }
 
   private renderSignInWithBrowser() {
-    const basicAuth = this.props.supportsBasicAuth
-    const browserSignInLink = (
-      <LinkButton
-        className="welcome-link-button link-with-icon"
-        onClick={this.signInWithBrowser}
-      >
-        Sign in using your browser
-        <Octicon symbol={OcticonSymbol.linkExternal} />
-      </LinkButton>
-    )
-
-    const browserSignInButton = (
-      <Button type="submit" onClick={this.signInWithBrowser}>
-        Sign in using your browser
-      </Button>
-    )
+    if (this.props.endpoint === getDotComAPIEndpoint()) {
```

Comment:
```suggestion
    // we don't render this here because the user will have already
    // had the option to sign in via the browser earlier in the sign-in flow
    if (this.props.endpoint === getDotComAPIEndpoint()) {
```